src/test/py/peer/peer_test.py

```diff
@@ -202,72 +202,238 @@ def test_verify_integrity(self):
         self.ledger.queue[0] = (self.peer, b"corrupted_hash")
         self.assertFalse(self.ledger.verify_integrity())
 
-    async def test_announce_trade(self):
+    async def test_handle_exchange_announcement_deadline_trade(self):
+        """
+        Test case for the handle_exchange_announcement_deadline method for the trade exchange_type
+        of the Peer class.
+        """
+
+        announcement_type = "trade"
+        announcement_key = "key"
+        offer_announcement = OfferAnnouncement("trade", 0, "artwork")
```

Comment:
Shouldn't the 3rd arg be trade since that arg is the `exchange_type`?

It's usually better to have the type be the a class that derives from something like `OfferAnnouncement` or, at a minimum have the arg be an enum like object. It's too easy to get strings, ints, etc wrong.